build: ci failures due to rate limiting

These lines: 

```
# Copy CA certificates store
COPY --from=public.ecr.aws/amazonlinux/amazonlinux:2 /etc/ssl /etc/ssl
COPY --from=public.ecr.aws/amazonlinux/amazonlinux:2 /etc/pki /etc/pki
```

Seem to get rate limited:

```
2021-09-17T11:49:57.2256657Z Step 8/12 : FROM scratch
2021-09-17T11:49:57.2257174Z  ---> 
2021-09-17T11:49:57.2258156Z Step 9/12 : COPY --from=public.ecr.aws/amazonlinux/amazonlinux:2 /etc/ssl /etc/ssl
2021-09-17T11:49:57.7072597Z 2: Pulling from amazonlinux/amazonlinux
2021-09-17T11:49:57.7923908Z invalid from flag value public.ecr.aws/amazonlinux/amazonlinux:2: toomanyrequests: Rate exceeded
2021-09-17T11:49:57.7956066Z make: *** [Makefile:28: controller-image] Error 1
2021-09-17T11:49:57.7965638Z ##[error]Process completed with exit code 2.
```

Interestingly our SDK does not have the same rate limit issue, so we can probably copy these cert bundles from the SDK instead.

--- a/controller/Dockerfile
+++ b/controller/Dockerfile
@@ -9,8 +9,9 @@ RUN cargo install --locked --target ${ARCH}-bottlerocket-linux-musl --path . --r
 
 FROM scratch
 # Copy CA certificates store
-COPY --from=public.ecr.aws/amazonlinux/amazonlinux:2 /etc/ssl /etc/ssl
-COPY --from=public.ecr.aws/amazonlinux/amazonlinux:2 /etc/pki /etc/pki
+COPY --from=build /etc/ssl /etc/ssl
+COPY --from=build /etc/pki /etc/pki
+# Copy binary
 COPY --from=build /src/controller/bin/controller ./
 
 ENTRYPOINT ["./controller"]

